array/code/ConferenceCall.js

```diff
@@ -0,0 +1,8 @@
+module.exports.function = function conferenceCall (contacts) {
+  // when linked contacts (input property in action) set max (Many), contacts is an array even if it has only ONE element
+  var rslt = "";
```

Comment:
we usually don't abbreviate like this in all other examples. just use `result` to be consistent.